src/api/spec/controllers/webui/attributes_spec.rb

```diff
@@ -16,4 +16,42 @@
       expect(flash[:error]).to eq("Project not found: Does:Not:Exist")
     end
   end
+
+  describe 'POST #create' do
+    let(:attribute_type_0) { create(:attrib_type_with_namespace, value_count: 0) }
+    let(:attribute_type_1) { create(:attrib_type_with_namespace, value_count: 1) }
+    let(:attribute_type_1_name) { "#{attribute_type_1.namespace}:#{attribute_type_1.name}" }
+
+    before do
+      login user
+    end
+
+    context 'with editable values' do
+      before do
+        post :create, params: { attrib: { project_id: user.home_project.id, attrib_type_id: attribute_type_1.id } }
```

Comment:
Why are we using `project_id` instead of `project` in the strong params? :rage: 

ok, we could change it but this is unrelated to the PR, so leave it as it is.